redfish-core/lib/assembly.hpp

```diff
@@ -286,7 +286,7 @@ void getAssemblyPresence(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
             if (fru == "panel0" || fru == "panel1")
             {
                 data["Oem"]["OpenBMC"]["@odata.type"] =
-                    "#OemAssembly.v1_0_0.Assembly";
```

Comment:
This is under the OpenBMC name space but an IBM schema?

Reply:
> This is under the OpenBMC name space but an IBM schema?

Ok, we can modify IBMAssembly schema - to rename `oem/OpenBMC/ReadyToRemove` to `oem/IBM/ReadyToRemove`.

Reply:
I'll keep them as they were originally `Oem/OpenBMC/ReadyToRemove `and put the schema file as `OpenBMCAssembly.json`